### PR TITLE
Prevent crash when deleting a job with high frequency schedule

### PR DESCRIPTION
**Pull request by richard-julien**
Try to solve https://github.com/victorcoder/dkron/issues/182.

### CODE REVIEW

File: dkron/queries.go [hunk 1]
```diff
@@ -25,6 +25,13 @@ func (a *AgentCommand) RunQuery(ex *Execution) {
 
 	job, _ := a.store.GetJob(ex.JobName)
 
+	//Job can be removed and the QuerySchedulerRestart not yet received.
+	//In this case, the job will be nil. Simply cancel the current execution
+	if job == nil {
+		log.Debug("Job was removed, cancelling this excution, waiting for QuerySchedulerRestart")
```
Comment (victorcoder) [inline dkron/queries.go]: I prefer to say "Job not found", but LGTM

File: dkron/queries.go [hunk 2]
```diff
@@ -25,6 +25,13 @@ func (a *AgentCommand) RunQuery(ex *Execution) {
 
 	job, _ := a.store.GetJob(ex.JobName)
```
Comment (victorcoder) [inline dkron/queries.go]: `err` should be checked here to see if it's `store.ErrKeyNotFound` then logging

Comment (richard-julien) [inline dkron/queries.go]: Agree that's a better way to do it. I will change that.

Comment (richard-julien) [inline dkron/queries.go]: PR updated.

**Comment by victorcoder** [comment]
@richard-julien You must run `go fmt` prior to commit, thanks!

**Comment by richard-julien** [comment]
Done. Sorry again ... devenv config updated to do that automatically.

**Pull request merged**